cfpb/cf-icons
# All new mixins and variables

All the mixins and variables. Most of these are directly from or heavily inspired by Font Awesome, so I also added attribution and a note to our license. 
## Additions
- Mixins for sizing, rotating, and animating icons
- Mixins for adding characters 
- Adding cf-icon-download to documentation to close #41 
- Adding attribution to dual license parts under Font Awesome's MIT licensing. 
## Review
- @Scotchester
- @ascott1 
- @contolini
- @anselmbradford 
## Testing
- Copy and branch and run `grunt`
- Check `docs/index.html` and `demo/index.html`
## Notes
- Closes #4. Closes #41
- I blame topdoc for all delays on this
